k8s-model-server/README.md

```diff
@@ -0,0 +1,170 @@
+# Kubernetes TensorFlow Model Server
+This repository contains docker files and manifests to deploy a generic 
+[TensorFlow model server](https://www.tensorflow.org/serving/) on a [Kubernetes](https://kubernetes.io/) cluster. 
+Rather them baking the served model into the image, as the TensorFlow inception example demonstrates, this 
+example demonstrates how to serve a model from distributed storage.  [GKE](https://cloud.google.com/kubernetes-engine/) 
+(Google Kubernetes Engine) and [GCS](https://cloud.google.com/storage/) (Google Cloud Storage) are used here, 
+but the example is trivially adaptable to an on-premises (on-prem) cluster using NFS (Network File System) or 
+HDFS (Hadoop Distributed File System), or an AWS (Amazon Web Services)cluster using S3 (Simple Storage Service).
+
+## Create Google Cloud Platform (GCP) Project 
+This tutorial uses [GCP](https://cloud.google.com/).
+[This documentation](https://cloud.google.com/resource-manager/docs/creating-managing-projects) demonstrates how to 
+create and manage a new Project on GCP. There is a [free tier](https://cloud.google.com/free/) available that you can 
+use to try it out.
+
+## Build a Model Server Docker Image
+The [docker file](docker/Dockerfile) builds a docker image containing default the TensorFlow model server tuned for 
+amd64 architecture. You can modify this file to support the architecture of your choice.
+
+### Set the User (Optional)
+Note the line at the beginning of the Dockerfile.
+
+```
+ENV MS_USER=model-server
+```
+
+This is the user that the model-server process will run as. You can modify this user to change the name. This user 
+will be used in the Kubernetes Deployment's PodSecurityContext so that the container is not run with root privileges
+(This is considered to be a best practice).
+
+### Build the Image
```

Comment:
Can we have a `summary` section in the beginning where we recommend using an image that we have already compiled? Compiling manually is most likely for different platform optimizations or for private registry reasons.

Reply:
Sure but where are we hosting that image. We can update later once the image gets pushed.

Reply:
gcr.io/kubeflow?